var/ramble/repos/builtin/modifiers/gcp-cloud-logging/modifier.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2022-2024 The Ramble Authors
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+from ramble.modkit import *  # noqa: F403
+
+
+class GcpCloudLogging(BasicModifier):
+    """Upload experiment logs to Google Cloud Platform Cloud Logging.
```

Comment:
Just curious, is cloud-logging something that gets configured upon cluster creation? Would we want to support some configurability in the modifier? One thing I may want to configure is how frequently the log gets uploaded, and/or if the logging agent has some knobs on resource control. Mostly wonder if there could be some performance hit when this is enabled.

Reply:
I haven't explored the performance impact yet, but I don't know of any user-side control of the cloud-logging agent.

I can research this some more, but this was mostly to support other workflows within our organization.

Reply:
I ran some performance impact studies which I can share with you on Monday, but this has no noticeable impact on the performance of the workloads I ran.

Reply:
Thanks for checking that! One last annoying question: do you see the need to verify the logging agent is actually up and running? I imagine something like `service google-fluentd status`, although I don't really know if that's the right agent service.

Reply:
Done. :)